IBX-1588: Added category select and blacklist update (#123)
ezsystems/user-documentation: docs/personalization/recommendation_models.md

#### Blacklist

Items from this list are not recommended in any scenario.

> A model can be configured manually by a user.  

Reviewer: ```suggestion
The model can be configured manually by a user.  
```

You can use this model to exclude test products or products that are used for system monitoring. 

> Added element will never be recommended so it must be treated with care, because the blacklist model applies to all scenarios that exist in the system.

Reviewer: ```suggestion
An element added to this list will never be recommended so it must be treated with care, because the blacklist model applies to all scenarios that exist in the system.
```

    If you do not want to group recommendations based on a certain attribute, 
    remove the attribute parameter from the request. 
    The submodel is then omitted.

## Segments

> Segments allow to get personalized content suitable for particular users groups. They compute models based on segment attribute factor.

Reviewer: ```suggestion
Segments allow getting personalized content suitable for particular user groups. They compute models based on segment attribute factor.
```

> Information with user segment are provided in each event which comes from the tracking script.

Reviewer: ```suggestion
Information with user segment is provided in each event which comes from the tracking script.
```

First, make sure you have [enabled personalization](https://doc.ibexa.co/en/latest/guide/personalization/enabling_personalization/#enable-personalization) and [configured item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).

> If your [user role](../site_organization/organizing_the_site.md#permissions) includes 

Reviewer: ```suggestion
If your [user Role](../site_organization/organizing_the_site.md#permissions) includes 
```

the `Segment/All functions`, `Segment group/All functions` Policies, you can configure segment settings in the models according to your requirements.
To do this, go to the **Models** section and click the **Edit** icon next to a name of the model.

> With segments groups you can assign users to different recommendation groups based on data gathered and deliver recommendations to these user groups.

Reviewer: ```suggestion
With segment groups you can assign users to different recommendation groups based on data gathered and deliver recommendations to these user groups.
```
